x-pack/platform/packages/shared/onechat/onechat-common/base/namespaces.ts

```diff
@@ -11,12 +11,17 @@
  */
 export const internalNamespaces = {
   platformCore: 'platform.core',
+  coreSecurity: 'core.security',
 } as const;
 
 /**
  * List of protected namespaces which can only be used by internal tools.
  */
-export const protectedNamespaces: string[] = [internalNamespaces.platformCore, 'observability'];
+export const protectedNamespaces: string[] = [
+  internalNamespaces.platformCore,
+  internalNamespaces.coreSecurity,
```

Comment:
@elastic/workchat-eng I had understood `core.security` to be the desired internal solution namespace pattern from the Protected Namespaces section of the planning [document](https://docs.google.com/document/d/10ZlcUZhrnxpWc6NDgv1YIXIUxc6AtNVhTpto_yQl9PU/edit?tab=t.l1c8mwk3gs0b). Should I just use `'security'` like o11y did?